src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -66,24 +68,36 @@ public OtlpExporterOptions()
 
         this.ApplyConfiguration(configuration, configurationType);
 
-        this.HttpClientFactory = this.DefaultHttpClientFactory = () =>
+        this.DefaultHttpClientFactory = () =>
         {
             return new HttpClient
             {
                 Timeout = TimeSpan.FromMilliseconds(this.TimeoutMilliseconds),
             };
         };
 
-        this.BatchExportProcessorOptions = defaultBatchOptions;
+        this.BatchExportProcessorOptions = defaultBatchOptions!;
     }
 
     /// <summary>
-    /// Gets or sets the target to which the exporter is going to send telemetry.
-    /// Must be a valid Uri with scheme (http or https) and host, and
-    /// may contain a port and path. The default value is
-    /// * http://localhost:4317 for <see cref="OtlpExportProtocol.Grpc"/>
-    /// * http://localhost:4318 for <see cref="OtlpExportProtocol.HttpProtobuf"/>.
+    /// Gets or sets the target to which the exporter is going to send
+    /// telemetry.
     /// </summary>
+    /// <remarks>
+    /// Notes:
+    /// <list type="bullet">
+    /// <item><see cref="Endpoint"/> must be a valid <see cref="Uri"/> with
+    /// scheme (http or https) and host, and may contain a port and path.</item>
+    /// <item>The default value is based on the <see cref="Protocol"/> property:
+    /// <list type="bullet">
+    /// <item><c>http://localhost:4317</c> for <see
+    /// cref="OtlpExportProtocol.Grpc"/>.</item>
+    /// <item><c>http://localhost:4318</c> for <see
```

Comment:
When using `HttpProtobuf`, for the default case, we append the signal name to the path. Could we update the documentation comments here to reflect that? https://github.com/open-telemetry/opentelemetry-dotnet/pull/5385#discussion_r1501264840

Reply:
How's this? https://github.com/open-telemetry/opentelemetry-dotnet/pull/5434/commits/57096cf11a0dba22ba0fd86cc6b75e6d3c1fae67